contrib/connector/nvmeof/nvmeof_helper.go

```diff
@@ -147,7 +147,7 @@ func Connect(connMap map[string]interface{}) (string, error) {
 	targetPortal := conn.TgtPortal
 	port := conn.TgtPort
 	nvmeTransportType := "rdma"
-	log.Printf("conn information:%s, %s, %s ", connNqn, targetPortal,  port)
+	log.Printf("conn information:%s, %s, %s ", connNqn, targetPortal, port)
```

Comment:
These changes are not related File share. Are these changes related to goimports ? then probably you can ignore these files as Lean has a PR https://github.com/opensds/opensds/pull/718 for these.

Reply:
mistake, need to remove

Reply:
Need to run go fmt then everything will be fine